tests/FSharp.Compiler.ComponentTests/EmittedIL/Inlining/Inlining.fs

```diff
@@ -41,6 +41,30 @@ module Inlining =
         |> getCompilation
         |> verifyCompilation
 
```

Comment:
Ditto:
    [<Theory; FileInlineData("FSharpDelegateBetaReduction.fs", Realsig=BooleanOptions.True, Optimize=BooleanOptions.Both)>]
    let ``FSharpDelegateBetaReduction_fs_Optimize`` compilation =
        compilation
        |> getCompilation
        |> withOptions [ "--test:EmitFeeFeeAs100001" ]
        |> asExe
        |> withEmbeddedPdb
        |> withEmbedAllSource
        |> ignoreWarnings
        |> verifyILBaseline

Now that I think about TrueAndFalse would have been a better name than both.